### PR TITLE
Correct PHP logs issued during admin observer's processing

### PR DESCRIPTION
**Pull request by scottcwilson**
--> PHP Warning: Undefined array key "fieldname" in /Users/scott/sites/gh_demo_158/admin/customers.php on line 695.

and a deprecation from HTMLSpecialChars when passed an empty arg 1.

### CODE REVIEW

**Comment by scottcwilson** [comment]
This is still a bug.

File: YOUR_ADMIN/includes/classes/observers/CustomerTaxExemptAdminObserver.php [hunk 1]
```diff
@@ -63,7 +63,11 @@ public function update(&$class, $eventID, $p1, &$p2, &$p3, &$p4, &$p5, &$p6)
                           LIMIT 1"
                     );
                     $customers_tax_exempt = ($current_value->EOF) ? '' : $current_value->fields['customers_tax_exempt'];
-                    $field_value = htmlspecialchars($customers_tax_exempt, ENT_COMPAT, CHARSET, true);
+                    if (!empty($customers_tax_exempt)) { 
+                        $field_value = htmlspecialchars($customers_tax_exempt, ENT_COMPAT, CHARSET, true);
+                    } else {
+                        $field_value = ''; 
+                    }
```
Comment (lat9) [inline YOUR_ADMIN/includes/classes/observers/CustomerTaxExemptAdminObserver.php]: It'd be simpler to simply cast the database value (which could be `NULL`) to a string.

**Review by lat9 — CHANGES_REQUESTED** [review]
Please simplify via casting.

File: YOUR_ADMIN/includes/classes/observers/CustomerTaxExemptAdminObserver.php (hunk 1 above)
Comment (scottcwilson) [reply YOUR_ADMIN/includes/classes/observers/CustomerTaxExemptAdminObserver.php]:
```
$field_value = htmlspecialchars((string)$customers_tax_exempt, ENT_COMPAT, CHARSET, true);
```
like that?

Comment (lat9) [reply YOUR_ADMIN/includes/classes/observers/CustomerTaxExemptAdminObserver.php]: That works!

**Comment by scottcwilson** [comment]
Updated.

**Review by lat9 — APPROVED** [review]

**Comment by lat9** [comment]
Contains corrections for issues #5 and #6.

**Pull request merged**